docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -40,6 +40,17 @@ You cannot update {dfanalytics-cap} configurations. Instead, delete the
 dedicated for {ml} processes. For general {ml} settings, see 
 {ref}/ml-settings.html[{ml-cap} settings in {es}].
 
+[float]
+[[dfa-time-limitations]]
+=== {dfanalytics-jobs-cap} runtime may vary
+
+The runtime of the {dfanalytics-jobs} depends on numerous factors, such as the 
+size of the dataset, the type of analytics, the number of fields that are included 
+in the analysis, the {ref}/ml-dfanalytics-resources.html#ml-hyperparameter-optimization[hyperparameters] used, and so on. For this reason, a 
+general runtime value that applies to all or most of the situations does not 
+exist. The runtime of a {dfanalytics-job} may take from a couple of minutes up 
+to tens of hours in extreme cases.
```

Comment:
There are three things I think we could improve in this paragraph:
1. While it's true that a general runtime does not exist, we can recommend that for datasets of greater than let's say 100 000 points, we recommend that users start with a low training percent and run a few jobs just to get a feel for how the runtime will scale with the increased number of datapoints and how their quality of results will scale with increased training percentage. 

@viglia 's experiment with the Rain in Australia dataset shows that with 140 000 datapoints, this dataset still has a runtime of less than 1 hour, so it would definitely be possible to run several training percentage experiments in a workday. 

2. In favour of transparency, I believe we should push the upper bound runtime from `ten hours` to at least 35 hours, which is something we have seen during testing. 

3. We should add a note that the general trend for runtime is to see it increase with increasing number of fields in a nearly linear fashion (provided that the number of datapoints analysed remains constant).